src/examples/hello_world/hello_world.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Field,
+  SmartContract,
+  state,
+  State,
+  Party,
+  method,
+  DeployArgs,
+  PrivateKey,
+} from '../../../dist/server';
+
+export const adminPrivateKey = PrivateKey.random();
+export const adminPublicKey = adminPrivateKey.toPublicKey();
+
+export class HelloWorld extends SmartContract {
+  @state(Field) x = State<Field>();
+
+  deploy(input: DeployArgs) {
+    super.deploy(input);
+    this.x.set(Field(3));
```

Comment:
Since we intend these tests/examples to be user-facing for others to see, we should switch the state initialization to its own function as previously discussed :D

Reply:
@MartinMinkov, I think the conclusion in our stand-up was that we would find a way to "automate" state initialization, e.g. based off of a default value the user passes to `State`. Using `deploy` until that happens seems fine to me

Reply:
+1 to @mitschabaude comment. The contract is based off of the example that @bkase and others have been presenting at meetups. We decided to leave the deploy method with the state initialization in the example until we can automate it.